Dockerfile-mysterium-node

```diff
@@ -1,15 +1,21 @@
 FROM ubuntu:latest
 MAINTAINER Valdas Petrulis <petrulis.valdas@gmail.com>
 
-# e.g. PACKAGE_FILE="build/package/mysterium-node_0.0.1_linux_amd64.deb"
-ARG PACKAGE_FILE
-COPY $PACKAGE_FILE /tmp/mysterium-node.deb
+ENV VERSION 0.0.6
+ENV NODE 321
 
 # Install packages
 RUN apt-get update \
-    && dpkg --install /tmp/mysterium-node.deb
-    && apt-get install --fix-broken \
+    && apt install -y wget openvpn iptables\
```

Comment:
I have checked it again, dpkg didn't install dependecies and give a error, so build will be failed.
Other way to use apt-get directly
`RUN apt-get update \
    && apt install -y wget\
    && wget https://github.com/MysteriumNetwork/node/releases/download/${VERSION}/mysterium-node_linux_amd64.deb \
    && apt-get install -y ./mysterium-node_linux_amd64.deb \
    && rm -rf /var/cache/apk/* \
    && rm mysterium-node_linux_amd64.deb`